### PR TITLE
FEAT: add gotify

### PR DESCRIPTION
**Pull request by LiquidITGuy**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

Capture d’écran 2022-09-11 à 20 26 32

### CODE REVIEW

File: public/v4/apps/gotify.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+captainVersion: 4
+services:
+  '$$cap_appname':
+    image: gotify/server:$$cap_version
+    volumes:
+      - '$$cap_appname-data:/app/data'
+      - '$$cap_appname-config:/etc/gotify/config.yml'
```
Comment (githubsaturn) [inline public/v4/apps/gotify.yml:7]: I don't think I've seen any volume names being mapped to a file. Typically they are mapped to the directory. Does this work just fine? I assume you've tested this?

Comment (matbgn) [reply public/v4/apps/gotify.yml:7]: Double this warning as all my tests with direct file map failed.

Comment (LiquidITGuy) [reply public/v4/apps/gotify.yml:7]: in https://gotify.net/docs/config we can see 
>gotify/server looks in the following paths for config files
./config.yml
/etc/gotify/config.yml

This file (even if not documented in docker file) is required if you want to customize your gotify

Comment (matbgn) [reply public/v4/apps/gotify.yml:7]: It's not about the config file need but its syntax.

Should it not be something like that instead:

- '$$cap_appname-config:/etc/gotify/'

Did you try your template with a fresh install on Caprover?

Comment (LiquidITGuy) [reply public/v4/apps/gotify.yml:7]: I just tried to redeploy with the same file on my pr and it seems to work.
I will try to change to follow the request as requested but I was thinking docker can mount a file as a volume if only the file need to be keeped.
Capture d’écran 2022-11-30 à 16 24 54

Comment (LiquidITGuy) [reply public/v4/apps/gotify.yml]: updated and tried ready @matbgn :)

File: public/v4/apps/gotify.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: gotify/server:$$cap_version
+        volumes:
+            - '$$cap_appname-data:/app/data'
+            - '$$cap_appname-config:/etc/gotify/'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
```
Comment (matbgn) [inline public/v4/apps/gotify.yml:10]: As really tiny advice I will more prefer a variable called cap_gotify_version but anyway seems to work, so... cool!!

File: public/v4/apps/gotify.yml (hunk 1 above)
Comment (matbgn) [reply public/v4/apps/gotify.yml]: Good job! I only have a tiny a best practice advice (see below) for maintenance purpose only but other than that well done!

File: public/v4/apps/gotify.yml (hunk 2 above)
Comment (LiquidITGuy) [reply public/v4/apps/gotify.yml:10]: thanks for the advice, I will do it next time :)

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**